src/nile/cli.py

```diff
@@ -43,7 +43,7 @@ def network_option(f):
 def _validate_network(_ctx, _param, value):
     """Normalize network values."""
     # normalize goerli
-    if "goerli" in value or "testnet" in value:
+    if "goerli" == value or "testnet" in value:
```

Comment:
```suggestion
    if "testnet" == value:
```
The check for equality to "goerli" is unnecessary because "goerli" is part of the NETWORKS. I would also remove the "in" check in favor of equality, because recognizing for example "testnet-2" as goerli, instead of goerli2 seems a little confusing.

Reply:
In most cases, I think the user will be passing values matching NETWORKS elements, so I would move this on top of this check, to avoid two ifs that won't enter in most cases. 

```
# check if value is accepted
if value in NETWORKS:
    return value
```